emu/src/bitwise.rs

```diff
@@ -9,9 +9,8 @@ pub trait Bits {
     fn set_bit_off(&mut self, bit_idx: u8);
     fn toggle_bit(&mut self, bit_idx: u8);
     fn set_bit(&mut self, bit_idx: u8, value: bool);
-    fn get_bit(self, bit_idx: u8) -> bool;
-    fn get_bits(self, bits_range: RangeInclusive<u8>) -> u32;
-    fn are_bits_on(self, bits_range: RangeInclusive<u8>) -> bool;
+    fn get_byte(&self, byte: u8) -> u8;
+    fn set_byte(&mut self, byte: u8, value: u8);
```

Comment:
I would change to something more clear like `byte_nth` or similar to clarify it is an index